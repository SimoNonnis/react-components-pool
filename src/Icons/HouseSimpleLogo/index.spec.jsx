/* eslint-env jest */
import React from 'react';

import HouseSimpleLogo from './';

describe('Test <HouseSimpleLogo /> icon', () => {
  it('should render HouseSimpleLogo icon correctly with default props', () => {
    const wrapper = shallow(<HouseSimpleLogo />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render HouseSimpleLogo icon correctly with props', () => {
    const wrapper = shallow(
      <HouseSimpleLogo fill="#a4a1a1" className="pof" height={30} width={30} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
