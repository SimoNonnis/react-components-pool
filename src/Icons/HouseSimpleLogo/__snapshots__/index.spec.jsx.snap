// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <HouseSimpleLogo /> icon should render HouseSimpleLogo icon correctly with default props 1`] = `
<svg
  className="houseSimpleLogo"
  height={31}
  viewBox="0 0 404.8 71.4"
  width={175}
  xmlns="http://www.w3.org/2000/svg"
>
  <title>
    HouseSimpleLogo Icon
  </title>
  <path
    className="houseSimpleLogoPath"
    d="M97.3,25.7v16c0,5.6,4.2,8.8,9.2,8.8,4.7,0,8.2-3.2,8.2-8.9V22.2a13.86,13.86,0,0,1,4.6,1.4,8.6,8.6,0,0,1,2.6,2.9V42.6c0,8.8-8.3,14.5-15.9,14.5S90,51.4,90,42.6V21.3s3.3.4,4.8,1.4A10.09,10.09,0,0,1,97.3,25.7Z"
    fill="#1E2938"
  />
  <path
    className="houseSimpleLogoPath"
    d="M127.8,46.3c2.5,3.4,5.9,4.4,9.9,4.4,2.5,0,5.9-1.1,5.9-4.1,0-7.1-14.9-5.7-14.9-16.8,0-6.6,5.1-9.5,11.2-9.5,5.6,0,8.5,2,8.5,2a5.61,5.61,0,0,1,1.3,3.6,7,7,0,0,1-1.3,4.2,9.06,9.06,0,0,0-7.4-3.3c-2.2,0-4.6.8-4.6,3.2,0,6.1,14.9,5,14.9,16,0,7.9-6.4,11.2-13.5,11.2S127.6,54,127.6,54a7.16,7.16,0,0,1-.9-3.3,14.85,14.85,0,0,1,.9-4.2v-.4Z"
    fill="#1E2938"
  />
  <path
    className="houseSimpleLogoPath"
    d="M187.9,54.2c0,.1-4.4,2.9-11.4,2.9-10.5,0-19.6-8-19.6-17.9a19.1,19.1,0,0,1,19.2-19c12.2,0,19.8,9.6,19.8,20.2a8,8,0,0,1-.1,1.1l-31.7.3c1.8,6.2,6.9,8.8,13.4,8.8A15.08,15.08,0,0,0,187.8,47a8.14,8.14,0,0,1,.9,3.9c.1,2.4-.8,3.3-.8,3.3m0-18.8c-1.2-5.2-6-8.6-11.5-8.6-6.1,0-10.4,3.2-12.2,8.6Z"
    fill="#1E2938"
  />
  <path
    className="houseSimpleLogoPath"
    d="M64.7,20.1c-10.5,0-19.4,8.5-19.4,18.4s8.9,18.4,19.4,18.4,19.4-8.6,19.4-18.4S75.3,20.1,64.7,20.1m0,29.6C58.4,49.7,53,44.6,53,38.6s5.4-11.1,11.7-11.1,11.7,5.1,11.7,11.1S71.1,49.7,64.7,49.7"
    fill="#1E2938"
  />
  <path
    className="houseSimpleLogoPath"
    d="M396.8,54.4c0,.1-4.4,2.9-11.4,2.9-10.6,0-19.6-8-19.6-17.9a19.1,19.1,0,0,1,19.2-19c12.2,0,19.8,9.6,19.8,20.2a8,8,0,0,1-.1,1.1L373,42c1.8,6.1,6.9,8.8,13.4,8.8a15.08,15.08,0,0,0,10.3-3.6,8.46,8.46,0,0,1,.9,3.8,6.51,6.51,0,0,1-.8,3.4m0-18.9c-1.2-5.2-6-8.6-11.5-8.6-6.1,0-10.4,3.2-12.2,8.6Z"
    fill="#1E2938"
  />
  <path
    className="houseSimpleLogoPath"
    d="M247.4,34.9c0-8.8,8.3-14.5,15.8-14.5A16.09,16.09,0,0,1,275.3,26a16,16,0,0,1,12.1-5.6c7.6,0,16,5.7,16,14.5V56.2a6.67,6.67,0,0,1-3.3.9,11.1,11.1,0,0,1-3.9-.9V35.9c0-5.6-3.9-8.8-9-8.8-4.3,0-8.2,3.6-8.2,7.8V56.2a10,10,0,0,1-3.6.9,9.31,9.31,0,0,1-3.6-.9V35.9c0-5.6-3.9-8.8-9-8.8a9,9,0,0,0-8.2,8.8V56.2a10.85,10.85,0,0,1-3.7.9,9.18,9.18,0,0,1-3.5-.9c.1-1.2,0-21.3,0-21.3"
    fill="#1E2938"
  />
  <path
    className="houseSimpleLogoPath"
    d="M309.8,35.9V29.5s4.8-9,16.9-9c11.2,0,19.7,7.5,19.7,19.1,0,10.5-8.6,17.8-18.5,17.8-4.2,0-7.9-.9-10.8-3.8V71.5a14.53,14.53,0,0,1-4.7-1.9,10.86,10.86,0,0,1-2.5-3.3V35.9M317,46.3c2,1,4.4,4,10.2,4,6.9,0,11.9-4.1,11.9-11.1,0-7.2-5.3-11.7-12.6-11.7-3.6,0-7.2,2.2-9.5,4.5Z"
    fill="#1E2938"
  />
  <path
    className="houseSimpleLogoPath"
    d="M352.7,56.2a6.23,6.23,0,0,0,3.6,1,6.08,6.08,0,0,0,3.5-1V15a9.12,9.12,0,0,0-2.6-3,9.55,9.55,0,0,0-4.4-1.3V56.2Z"
    fill="#1E2938"
  />
  <path
    className="houseSimpleLogoPath"
    d="M233.7,56.2a7.26,7.26,0,0,0,7,0V27.1s.4-1.7-2.3-3.8-4.8-1.4-4.8-1.4V56.2Z"
    fill="#1E2938"
  />
  <path
    className="houseSimpleLogoPath"
    d="M203.9,46.5c2.5,3.4,5.9,4.4,9.9,4.4,2.5,0,5.9-1.1,5.9-4.1,0-7.1-14.9-5.7-14.9-16.8,0-6.6,5.1-9.5,11.2-9.5,5.6,0,8.5,2,8.5,2a6.22,6.22,0,0,1,1.3,3.8,7.75,7.75,0,0,1-1.3,4.1,9.06,9.06,0,0,0-7.4-3.3c-2.2,0-4.6.8-4.6,3.2,0,6.1,14.9,5,14.9,16,0,7.8-6.4,11.2-13.5,11.2s-10.2-3.2-10.2-3.2a6.66,6.66,0,0,1-.9-3.8,6.11,6.11,0,0,1,.9-3.7v-.4Z"
    fill="#1E2938"
  />
  <path
    className="houseSimpleLogoPath"
    d="M8,33.1H6.5v23a11.1,11.1,0,0,0,3.9.9,12.93,12.93,0,0,0,4.1-.9V36.9A7.73,7.73,0,0,0,8,33.1Z"
    fill="#1E2938"
  />
  <path
    className="houseSimpleLogoPath"
    d="M37.1,27.6l-20.9-10a3.18,3.18,0,0,1-1.3-1.1,4.69,4.69,0,0,1-.4-1.9V4.9C13.2.7,7.6,0,7.6,0H6.5V19.6A2.85,2.85,0,0,0,7,21.5a7.74,7.74,0,0,0,1.2,1L28.5,32a5,5,0,0,1,1.8,1.4,4.11,4.11,0,0,1,.4,2.1V56.2a9.45,9.45,0,0,0,3.7.9,15,15,0,0,0,4.3-.9V30A2.76,2.76,0,0,0,37.1,27.6Z"
    fill="#1E2938"
  />
</svg>
`;

exports[`Test <HouseSimpleLogo /> icon should render HouseSimpleLogo icon correctly with props 1`] = `
<svg
  className="pof"
  height={30}
  viewBox="0 0 404.8 71.4"
  width={30}
  xmlns="http://www.w3.org/2000/svg"
>
  <title>
    HouseSimpleLogo Icon
  </title>
  <path
    className="houseSimpleLogoPath"
    d="M97.3,25.7v16c0,5.6,4.2,8.8,9.2,8.8,4.7,0,8.2-3.2,8.2-8.9V22.2a13.86,13.86,0,0,1,4.6,1.4,8.6,8.6,0,0,1,2.6,2.9V42.6c0,8.8-8.3,14.5-15.9,14.5S90,51.4,90,42.6V21.3s3.3.4,4.8,1.4A10.09,10.09,0,0,1,97.3,25.7Z"
    fill="#a4a1a1"
  />
  <path
    className="houseSimpleLogoPath"
    d="M127.8,46.3c2.5,3.4,5.9,4.4,9.9,4.4,2.5,0,5.9-1.1,5.9-4.1,0-7.1-14.9-5.7-14.9-16.8,0-6.6,5.1-9.5,11.2-9.5,5.6,0,8.5,2,8.5,2a5.61,5.61,0,0,1,1.3,3.6,7,7,0,0,1-1.3,4.2,9.06,9.06,0,0,0-7.4-3.3c-2.2,0-4.6.8-4.6,3.2,0,6.1,14.9,5,14.9,16,0,7.9-6.4,11.2-13.5,11.2S127.6,54,127.6,54a7.16,7.16,0,0,1-.9-3.3,14.85,14.85,0,0,1,.9-4.2v-.4Z"
    fill="#a4a1a1"
  />
  <path
    className="houseSimpleLogoPath"
    d="M187.9,54.2c0,.1-4.4,2.9-11.4,2.9-10.5,0-19.6-8-19.6-17.9a19.1,19.1,0,0,1,19.2-19c12.2,0,19.8,9.6,19.8,20.2a8,8,0,0,1-.1,1.1l-31.7.3c1.8,6.2,6.9,8.8,13.4,8.8A15.08,15.08,0,0,0,187.8,47a8.14,8.14,0,0,1,.9,3.9c.1,2.4-.8,3.3-.8,3.3m0-18.8c-1.2-5.2-6-8.6-11.5-8.6-6.1,0-10.4,3.2-12.2,8.6Z"
    fill="#a4a1a1"
  />
  <path
    className="houseSimpleLogoPath"
    d="M64.7,20.1c-10.5,0-19.4,8.5-19.4,18.4s8.9,18.4,19.4,18.4,19.4-8.6,19.4-18.4S75.3,20.1,64.7,20.1m0,29.6C58.4,49.7,53,44.6,53,38.6s5.4-11.1,11.7-11.1,11.7,5.1,11.7,11.1S71.1,49.7,64.7,49.7"
    fill="#a4a1a1"
  />
  <path
    className="houseSimpleLogoPath"
    d="M396.8,54.4c0,.1-4.4,2.9-11.4,2.9-10.6,0-19.6-8-19.6-17.9a19.1,19.1,0,0,1,19.2-19c12.2,0,19.8,9.6,19.8,20.2a8,8,0,0,1-.1,1.1L373,42c1.8,6.1,6.9,8.8,13.4,8.8a15.08,15.08,0,0,0,10.3-3.6,8.46,8.46,0,0,1,.9,3.8,6.51,6.51,0,0,1-.8,3.4m0-18.9c-1.2-5.2-6-8.6-11.5-8.6-6.1,0-10.4,3.2-12.2,8.6Z"
    fill="#a4a1a1"
  />
  <path
    className="houseSimpleLogoPath"
    d="M247.4,34.9c0-8.8,8.3-14.5,15.8-14.5A16.09,16.09,0,0,1,275.3,26a16,16,0,0,1,12.1-5.6c7.6,0,16,5.7,16,14.5V56.2a6.67,6.67,0,0,1-3.3.9,11.1,11.1,0,0,1-3.9-.9V35.9c0-5.6-3.9-8.8-9-8.8-4.3,0-8.2,3.6-8.2,7.8V56.2a10,10,0,0,1-3.6.9,9.31,9.31,0,0,1-3.6-.9V35.9c0-5.6-3.9-8.8-9-8.8a9,9,0,0,0-8.2,8.8V56.2a10.85,10.85,0,0,1-3.7.9,9.18,9.18,0,0,1-3.5-.9c.1-1.2,0-21.3,0-21.3"
    fill="#a4a1a1"
  />
  <path
    className="houseSimpleLogoPath"
    d="M309.8,35.9V29.5s4.8-9,16.9-9c11.2,0,19.7,7.5,19.7,19.1,0,10.5-8.6,17.8-18.5,17.8-4.2,0-7.9-.9-10.8-3.8V71.5a14.53,14.53,0,0,1-4.7-1.9,10.86,10.86,0,0,1-2.5-3.3V35.9M317,46.3c2,1,4.4,4,10.2,4,6.9,0,11.9-4.1,11.9-11.1,0-7.2-5.3-11.7-12.6-11.7-3.6,0-7.2,2.2-9.5,4.5Z"
    fill="#a4a1a1"
  />
  <path
    className="houseSimpleLogoPath"
    d="M352.7,56.2a6.23,6.23,0,0,0,3.6,1,6.08,6.08,0,0,0,3.5-1V15a9.12,9.12,0,0,0-2.6-3,9.55,9.55,0,0,0-4.4-1.3V56.2Z"
    fill="#a4a1a1"
  />
  <path
    className="houseSimpleLogoPath"
    d="M233.7,56.2a7.26,7.26,0,0,0,7,0V27.1s.4-1.7-2.3-3.8-4.8-1.4-4.8-1.4V56.2Z"
    fill="#a4a1a1"
  />
  <path
    className="houseSimpleLogoPath"
    d="M203.9,46.5c2.5,3.4,5.9,4.4,9.9,4.4,2.5,0,5.9-1.1,5.9-4.1,0-7.1-14.9-5.7-14.9-16.8,0-6.6,5.1-9.5,11.2-9.5,5.6,0,8.5,2,8.5,2a6.22,6.22,0,0,1,1.3,3.8,7.75,7.75,0,0,1-1.3,4.1,9.06,9.06,0,0,0-7.4-3.3c-2.2,0-4.6.8-4.6,3.2,0,6.1,14.9,5,14.9,16,0,7.8-6.4,11.2-13.5,11.2s-10.2-3.2-10.2-3.2a6.66,6.66,0,0,1-.9-3.8,6.11,6.11,0,0,1,.9-3.7v-.4Z"
    fill="#a4a1a1"
  />
  <path
    className="houseSimpleLogoPath"
    d="M8,33.1H6.5v23a11.1,11.1,0,0,0,3.9.9,12.93,12.93,0,0,0,4.1-.9V36.9A7.73,7.73,0,0,0,8,33.1Z"
    fill="#a4a1a1"
  />
  <path
    className="houseSimpleLogoPath"
    d="M37.1,27.6l-20.9-10a3.18,3.18,0,0,1-1.3-1.1,4.69,4.69,0,0,1-.4-1.9V4.9C13.2.7,7.6,0,7.6,0H6.5V19.6A2.85,2.85,0,0,0,7,21.5a7.74,7.74,0,0,0,1.2,1L28.5,32a5,5,0,0,1,1.8,1.4,4.11,4.11,0,0,1,.4,2.1V56.2a9.45,9.45,0,0,0,3.7.9,15,15,0,0,0,4.3-.9V30A2.76,2.76,0,0,0,37.1,27.6Z"
    fill="#a4a1a1"
  />
</svg>
`;
